src/main/java/build/buildfarm/instance/shard/JedisClusterFactory.java

```diff
@@ -186,12 +198,13 @@ private static Supplier<JedisCluster> createJedisClusterFactory(
             /* soTimeout=*/ Integer.max(2000, timeout),
             Integer.max(5, maxAttempts),
             password,
-            poolConfig);
+            identifier,
+            poolConfig,
+            false);
```

Comment:
nit: please incorporate the following
```suggestion
            /* ssl=*/ false);
```

Reply:
Fixed via 169bea9ac46ac69f699377a6475794e9d64b54e4 🚀